### ISSUE TITLE
Create lists

### ISSUE THREAD

**Issue by axelson** [IonicaBizau/medium-editor-markdown#4]
How is it possible to start a list?

Trying to create one on the demo site:
http://ionicabizau.github.io/medium-editor-markdown/

I would expect that beginning a line with '*' would work, but it doesn't.

**Comment by IonicaBizau** [comment]
It should work. I modified the HTML using the developer tools and this is the result:

![image](https://cloud.githubusercontent.com/assets/2864371/6958751/e566d600-d918-11e4-9691-ee04c1ba799b.png)

Also, you may want to add the [`orderedlist` or `unorderedlist` buttons](https://github.com/daviferreira/medium-editor#extra-buttons).

**Comment by axelson** [comment]
Okay, so with the default buttons there is no way for a user to start a list?

I've successfully added the unordered list button but after typing some text then pressing the ordered list button on the third line I've typed in  (without selecting any text before hand since I'm using the static toolbar) I get a `<span>` in the markdown output.

Here is what my editor looks like:
![screenshot 2015-04-02 17 11 31](https://cloud.githubusercontent.com/assets/9973/6977541/5c35448c-d95b-11e4-9157-230dacdb6b51.png)

Here is the markdown I've received for that:

```
Some text

Generally seems to work

*   <span style="line-height: 1.42857143;">Add a list</span>
```

Instead I expect the following markdown:

```
Some text

Generally seems to work

*   Add a list
```

I'm using medium-editor 4.1.1 and medium-editor-markdown 1.1.0.

Here is how I'm initializing medium-editor:

<pre>
        var options = {
            staticToolbar: true,
            buttons: ['bold', 'italic', 'underline', 'orderedlist', 'unorderedlist', 'anchor', 'header1', 'header2', 'quote'],
            extensions: {
                markdown: new MeMarkdown(function (receivedMarkdown) {
                    console.log("got markdown: " + receivedMarkdown);
                    markdown = receivedMarkdown;
                }),
            }
        };
        var editor = new MediumEditor('.md-editable', options);
</pre>

Am I missing something or is this a bug?

**Comment by IonicaBizau** [comment]
Yes, it's a bug. :+1: 

However, this is because of the [`to-markdown` library](https://github.com/domchristie/to-markdown). We're using `0.0.3` here, but I see some recent updates and `1.0.0-rc2`. 

The first step would be to upgrade to the latest version and rebuild the dist files.

**Comment by IonicaBizau** [comment]
@axelson Check if this issue is fixed in the `to-markdown-1.2.0` branch. The [demo website](http://ionicabizau.github.io/medium-editor-markdown/) already uses the newest version. For reference, see #5.

**Comment by IonicaBizau** [comment]
After some tests, I see that the conversion is now smoother. :star2: :dizzy:
